Add known caveats for Python/Ruby DogStatsD clients in detailed info docs (#11245)
DataDog/documentation: content/en/developers/dogstatsd/high_throughput.md

The Datadog Agent performs most optimally if the DogStatsD clients send packets the size of the `dogstatsd_buffer_size`. The packets must not be larger than the buffer size, otherwise, the Agent won't be able to load them completely in the buffer and some of metrics will be malformed. Use the corresponding configuration field in your DogStatsD clients.

<div class="alert alert-info">

>   <strong>Note for UDP</strong>: since UDP packets usually go through the Ethernet and IP layer, avoid IP packets fragmentation by limiting the packet size to a value lower than a single Ethernet frame on your network. Most of the time, IPv4 networks are configured with a MTU of 1500 bytes, so in this situation the packet size of sent packets should be limited to 1472.

Reviewer: ```suggestion
  <strong>Note for UDP</strong>: Because UDP packets usually go through the Ethernet and IP layer, you can avoid IP packets fragmentation by limiting the packet size to a value lower than a single Ethernet frame on your network. Most of the time, IPv4 networks are configured with a MTU of 1500 bytes, so in this situation the packet size of sent packets should be limited to 1472.
```

</div>

<div class="alert alert-info">

>   <strong>Note for UDS</strong>: for the best performances, UDS packet should have a size of 8192 bytes.

Reviewer: Is this saying the UDS packet size should be exactly 8192 bytes, or is this a value that it should be less than?

Author: @jtappa It should be exactly 8192 bytes I think as it is the general max size that the server will accept per packet over UDS. While smaller values are allowed, they won't be as performant since it would take more request to send the same amount of data.

Reviewer: ```suggestion
  <strong>Note for UDS</strong>: for the best performances, the UDS packet size should be 8192 bytes.
```

Reviewer: So in that case, I'd suggest just small modification to this sentence ^^
